include/deal.II/distributed/solution_transfer.h

```diff
@@ -157,17 +157,17 @@ namespace parallel
      *
      * Since data on DoFHandler objects with hp capabilities is associated with
      * many different FiniteElement objects, each cell's data has to be
-     * processed with its corresponding `future_fe_index`. Further, if
+     * processed with its corresponding `future fe index`. Further, if
```

Comment:
You're referencing a technical term here, which is used in function and variable names. I think it should be written with underscores if you mark it up with the backticks. If you want spaces, remove the backticks.

Reply:
Same everywhere else